### ISSUE TITLE
Extend paper buttons row alignment issue on 2022.6b0

### ISSUE THREAD

**Issue by krazos** [jcwillox/lovelace-paper-buttons-row#90]
After upgrading to 2022.6b0, buttons added to an entities row via `extend_paper_buttons_row` are not aligned properly for sensor entities, pushing the state of the sensor entity to the next line:

![Screenshot 2022-05-26 6 56 35 AM](https://user-images.githubusercontent.com/18428022/170475972-f2ad9895-a044-4c25-b99e-93f0a5dca6fc.png)

```yaml
type: entity-filter
entities:
  - entity: sensor.curbside_collection
    state_filter:
      - operator: '=='
        value: today
      - operator: '=='
        value: tomorrow
    extend_paper_buttons_row:
      position: center
      base_config:
        styles:
          button:
            margin-right: 8px
      buttons:
        - entity: sensor.curbside_collection
          icon: mdi:check-circle
          name: false
          tap_action:
            action: call-service
            service: garbage_collection.collect_garbage
            service_data:
              entity_id: sensor.curbside_collection
```

For what it's worth, substantially similar code still works properly for switch entities with a toggle switch:

![Screenshot 2022-05-26 7 09 07 AM](https://user-images.githubusercontent.com/18428022/170476774-fba7d84b-0820-4573-9a8f-61712bee1050.png)

**Comment by jcwillox** [comment]
Thanks for noticing this early, I've released [1.0.1](https://github.com/jcwillox/lovelace-paper-buttons-row/releases/tag/1.0.1) which should fix this.

**Comment by krazos** [comment]
Many thanks for the quick fix. That appears to have done the trick!
